### ISSUE TITLE
You could use a file chooser

### ISSUE THREAD

**Issue by Bunkerbewohner** [archeaopteryx/CASnumber_lookup#1]
https://github.com/archeaopteryx/CASnumber_lookup/blob/8cc21e9840e1b4890e46d43e3899c1548b681f5e/App.java#L91

To save your mum from having to put the .jar and the excel sheet in the same directory, you could just use a file chooser: https://docs.oracle.com/javase/tutorial/uiswing/components/filechooser.html

You also remember the chosen file, in case it's always the same, by saving the path (and any other options you want to remember) to a text file in Window's AppData folder, e.g.:

```
string appDataDir = System.getenv("APPDATA") + "\CASnumber_lookup";
string settingsPath = appDataDir + "\settings.txt";

// read/write settings from that file, it will persist
```

**Comment by archeaopteryx** [comment]
No, you're right. I like the suggestion of using the AppData :) Just had to hunt down some NPEs and try to handle bad input gracefully first.
Thanks again for looking it over!

**Comment by archeaopteryx** [comment]
And no, she shouldn't use an empty one every time ^^ That wouldn't be very useful

**Comment by archeaopteryx** [comment]
ok, added a settings file with default location and update option.

**Comment by Bunkerbewohner** [comment]
Nice!
